tools/clang/test/SemaHLSL/attributes/wavesize_range_valid.hlsl

```diff
@@ -0,0 +1,31 @@
+// RUN: %dxc -T lib_6_8 %s | FileCheck %s
+
+// Check the WaveSize attribute emits the right metadata in DXIL
```

Comment:
As a `SemaHLSL` test, I would have expected a check for -ast-dump only instead of end-to-end compilation.

For end-to-end compilation, we need to put the test somewhere under HLSLFileCheck where we can avoid running the test when the current validator doesn't support SM 6.8.

Reply:
I moved the test to a place that makes sense for end-to-end compilation.